retrofit/lib/annotations/annotations.dart

```diff
@@ -2,5 +2,6 @@ export 'requests.dart';
 export 'params.dart';
 
 class GenApiClient {
-  const GenApiClient();
+  final String path;
+  const GenApiClient([this.path = null]);
```

Comment:
Please change it to named optional parameter.

```dart
const GenApiClient({this.path});
```

Reply:
Done :)